app/charts/map/map.tsx

```diff
@@ -473,24 +501,26 @@ const useStyles = makeStyles<Theme>((theme) => ({
   },
 }));
 
-const ZoomButtons = ({
+const MapControlButtons = ({
+  refresh,
   zoomIn,
   zoomOut,
 }: {
+  refresh: () => void;
   zoomIn: () => void;
   zoomOut: () => void;
 }) => {
   const classes = useStyles();
 
   return (
-    <Box className={classes.zoomButtons}>
-      <ZoomButton iconName="add" handleClick={zoomIn} />
-      <ZoomButton iconName="minus" handleClick={zoomOut} />
+      <MapControlButton iconName="circle" handleClick={refresh} />
+      <MapControlButton iconName="add" handleClick={zoomIn} />
+      <MapControlButton iconName="minus" handleClick={zoomOut} />
     </Box>
   );
 };
```

Comment:
Nit: This MapControlButtons is not very useful IMHO. Better to have the buttons directly in the parent, this way we do not have to drill props.